### PR TITLE
When no order is specified, the order by indicator(triangle) in column header should be Cleared

### LINKED ISSUE #151: Bug: When no order is specified, the order by indicator(triangle) still indicate ASC on column
When clicking on a column header a third time, the order by field in request is cleared, but the column header still indicate the column is sorted by ASC order.

### PR DESCRIPTION
**Pull request by Haocen**
closes: https://github.com/shlinkio/shlink-web-client/issues/151

### CODE REVIEW

File: src/short-urls/ShortUrlsList.js [hunk 1]
```diff
@@ -50,9 +50,22 @@ const ShortUrlsList = (ShortUrlsRow) => class ShortUrlsList extends React.Compon
...
+    switch (this.state.orderDir) {
+      case 'ASC':
+        icon = caretUpIcon;
+        break;
+      case 'DESC':
+        icon = caretDownIcon;
+        break;
+      default:
+        return null;
+    }
```
Comment (acelaya) [inline src/short-urls/ShortUrlsList.js:55]: I have never been a big fan of `switch` statements or using `let` if it is not absolutely unavoidable :sweat_smile:

In this case I think we can simplify it. What do you think about this other approach:

```js
if (!this.state.orderDir) {
  return null;
}

const icon = this.state.orderDir === 'ASC' ? caretUpIcon : caretDownIcon;

return <FontAwesomeIcon icon={icon} className="short-urls-list__header-icon" />;
```

**Comment by acelaya** [comment]
I added a small comment to your PR.

Other than that, could you also add an entry to the changelog noticing this bugfix and linking to the issue number #151?

It should be added in the `[Unreleased]` block (which is at the top of the file), under the `Fixed` section, replacing the `* Nothing` by a link to the issue and an explanation of what has been fixed.

**Comment by acelaya** [comment]
~~Oh, and it would be awesome to have a test that covers this use case, but only if you have the time. I can add it later otherwise. Up to you.~~

Never mind. I have just realized there's not test at all for this component :man_facepalming:, so don't waste your time.

**Comment by Haocen** [comment]
@acelaya Well, guess I just made one for it. Slowing down the pipeline a little, if you don't mind.

**Comment by acelaya** [comment]
Awesome!

Thank you. This is much appreciated 😁

**Pull request merged**